### PR TITLE
Add dependency and update README

### PR DESCRIPTION
**Pull request by apiraino**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -49,7 +49,8 @@ RUN dpkg --add-architecture i386 && \
         libasound2-dev \
         libc6:i386 \
         libstdc++6:i386 \
-        libsdl1.2-dev
+        libsdl1.2-dev \
+        xutils-dev
```
Comment (apiraino) [inline Dockerfile:54]: I don't remember anymore why I added this. I'll update this comment when I have figured that out.

**Review by apiraino — COMMENTED** [review]
A round of updates and small docs improvements

File: Dockerfile (hunk 1 above)
Comment (apiraino) [reply Dockerfile:54]: ok found it. I need this for `lndir`

**Comment by apiraino** [comment]
@MBeijer I think we're done here, what do you think?

**Comment by MBeijer** [comment]
pretty much, as long as it doesn't break anything else. :D Size tweaking could be done though. :)

**Review by MBeijer — APPROVED** [review]

**Pull request merged**